helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/AvroSchemaVerifier.java

```diff
@@ -25,12 +24,15 @@
  */
 public class AvroSchemaVerifier {
   private final static AvroSchemaVerifier INSTANCE = new AvroSchemaVerifier();
+  private  static boolean isAvroVersionSupported = false;
 
   /**
    * Retrieves the verifier instance
    * @return
    */
   public static AvroSchemaVerifier get() {
+    AvroVersion avroVersion = AvroCompatibilityHelper.getRuntimeAvroVersion();
+    isAvroVersionSupported = avroVersion != AvroVersion.AVRO_1_10 && avroVersion != AvroVersion.AVRO_1_9;
```

Comment:
there are methods to compare avro version enums for laterThan etc. probably better here? (you want avro < 1.9)

Reply:
also, you want to make this call once, in static initialization ?

Reply:
and you want a comment explaining why and a TODO to fix this

Reply:
Hi Radai,
 I have addressed the comments, please take a look.